### ISSUE TITLE
[FEAT] Using a tag to denote separating between question and answer

### ISSUE THREAD

**Issue by richardrl** [st3v3nmw/obsidian-spaced-repetition#651]
Right now, the format for a flashcard must be:

Question
?
Answer

This really restricts the formatting we can have. Notably, we are not allowed to have line breaks.

For example,
Question
?
Answer

Answer line

Paragraph

This makes it hard to format our answers

**Comment by richardrl** [comment]
@AB1908 You can use multiple lines but you can't use white spaces. The current plugin ignores everything after a whitespace. Not having white spaces severely degrades the readability of your notes.

**Comment by richardrl** [comment]
@AB1908 But the power of Obsidian + SRS is that your cards can be made directly off note snippets. That saves a ton of work versus maintaining a separated anki deck + notes

**Comment by AB1908** [comment]
Yep I agree, I'm sure that's why we have so many SRS users haha.
